packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1120,3 +1132,68 @@ export async function pushDataToQueue(data: UploadType, requestQueueHandler: Req
 }
 
 export const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms))
+
+export abstract class BStackLogger {
```

Comment:
Can we use extensions on `logLevel` instead of handling it on our own? We don't need to manually handle the color coding and defining separate methods

Reply:
Closed offline